Mathlib/Data/List/Range.lean

```diff
@@ -27,15 +27,6 @@ namespace List
 
 variable {α : Type u}
 
-set_option linter.deprecated false in
-@[simp]
-theorem nthLe_range' {n m step} (i) (H : i < (range' n m step).length) :
-    nthLe (range' n m step) i H = n + step * i := get_range' i H
```

Comment:
I think these can't be deleted yet: they need to be `@[deprecated]` first.

Reply:
The continued existence of `nthLe` and other lemmas depending on it is an embarrassment to mathlib and the sooner they are purged the better. I don't think we can afford to wait a whole year to deprecate them.